CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+## v2.0.0-beta.58
```

Comment:
not specific to this PR, but we can drop the beta tag off of here...this was something we had very long ago for good reasons that just never got dropped. 🙃